### PR TITLE
Word

### LINKED ISSUE #2646: [JS Games Project] Word Scramble
# **Word Scramble Game**

## **Description 📃**

- Word Scramble Game challenge players to rearrange scrambled letters to form meaningful words. The game is a fun way to improve your vocabulary and spelling skills. The game is suitable for all age groups, especially for children and English learners.

## **Functionalities 🎮**

Game consists of following functionalities :

<ol>
<li>User will get a chance to select the difficulty level of the game(Easy,Medium or Hard).Based on the level of game, the number of guesses would change.</li>
<li>User has to rearrange the alphabets within given time interval only</li>
<li>For every correct answer, user would get 1 mark.</li>
<li>User can even refresh the word, if he/she is unable to get the answer.</li>
</ol>

## **How to Play? 🕹️**

- You have to select the difficulty level.
- After selecting the difficulty level, you would be getting the time interval to guess the word using the alphabets.
- At end of the game, you would be getting the score after finishing all the 10 questions accurately.

## **Working Video 📹**

https://github.com/ssitvit/Games-and-Go/assets/83013408/3aac8b67-6c8b-4eff-bfe6-877fe99448de

### Additional information

I kindly request to be assigned to it under GSSoC '23. I am excited to contribute and work on it.
@MohitGupta121 @pranjay-poddar

### PR DESCRIPTION
**Pull request by manasi-jain**
# Title and Issue number 

Word Scramble Game 

Close #2646

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@manasi-jain if this only Word Scramble project then why you adding flutter projects also in your PR. Remove all unnecessary folder and codes.

**Comment by manasi-jain** [comment]
@MohitGupta121 I have made changes to address the feedback provided. Please take a look at the updated code.

File: Js-Games/WordScramble/background.png
Comment (MohitGupta121) [inline Js-Games/WordScramble/background.png:1]: @manasi-jain please use don't upload image in PR use them from Link in your project.

File: Js-Games/WordScramble/word_scramble.gif
Comment (MohitGupta121) [inline Js-Games/WordScramble/word_scramble.gif:1]: @manasi-jain same for this files use the link. Upload these file somewhere and load them through link in your actual project.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@manasi-jain PTAL, Thanks.

**Pull request closed without merging**